plugins/wasi_nn/wasinn_ggml.cpp

```diff
@@ -1762,7 +1763,13 @@ Expect<ErrNo> setInput(WasiNNEnvironment &Env, uint32_t ContextId,
   if (Base64ImagePos.has_value() || CxtRef.Conf.ImagePath != ""sv) {
     // Prompt with image input. Check is llava or mllama case.
 
-    // First check the projection model is loaded.
+    // First check the projection model is given.
+    if (GraphRef.MMProjModelPath == ""sv) {
+      RET_ERROR(ErrNo::InvalidArgument,
+                "setInput: projection model path is not given."sv)
```

Comment:
I think we should also have a reasoning output, such as, "The given model does not support image input."

Reply:
I just updated it to `setInput: the given model does not support image input, so a projection model is required.`